application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -2972,14 +2978,24 @@ public function get_uploaded_files($sSessionKey, $iSurveyID, $sToken)
      * @param string $sPassword password
      * @return bool
      */
-    protected function _doLogin($sUsername, $sPassword)
+    protected function _doLogin($sUsername, $sPassword,$sPlugin)
```

Comment:
Partly looks just like in https://github.com/LimeSurvey/LimeSurvey/pull/836, why didn't you add your commits to it?

Reply:
If that's your preferred way, fine. However you as a contributor were probably able to edit the pull request to your needs whilst now I can not contribute but only comment lines or create pull requests for your fork, by creating a fork of your fork 👍